### ISSUE TITLE
Improve clarity of upload/download rates for other device

### ISSUE THREAD

**Issue by canton7** [syncthing/syncthing#2743]
The "Upload Rate" and "Download Rate" for the other devices in the GUI are confusing. The numbers show how much data we're download from / uploading to that device, but the layout implies that the numbers relate to that device's total upload/download rate.

@calmh suggested, from IRC:

```
<calmh> the one thing i could see is to remove the rates altoghether from the individual device section
<calmh> and have something more detailed on our own device. like: (ascii art follows)
<calmh> - Downloading: 1.17 Mbps
<calmh> +-- From ABCD123: 1.10 Mbps
<calmh> +-- From CBDE565: 0.07 Mbps
```

I'm happy to take this on when I get some free time: opening an issue so we don't forget.

**Comment by generalmanager** [comment]
Just for reference: https://github.com/syncthing/syncthing/issues/884 #821

**Comment by canton7** [comment]
Yeah, those issues are the previous iterations of trying to make this clearer, but I still get confused by it... I like @calmh's suggestion here - that's the first time this all made sense.

**Comment by generalmanager** [comment]
I agree and added my reasons in an edit (in case you answered via mail). The current implementation is only clear to me because by now it's learned behaviour to interpret everything as "local machine"-centric.

**Comment by Sophist-UK** [comment]
I was going to raise this as an issue myself, but I found this already existed.

I would like to add that "Upload" and "Download" imply some sort of hierarchical relationship when SyncThing is a P2P protocol where every node is equal in status.

Can I suggest that we replace "Download" with "Receiving" and "Upload" with "Sending"?
